app/javascript/components/patient/laboratory/FirstPositiveLaboratory.js

```diff
@@ -33,16 +34,22 @@ class FirstPositiveLaboratory extends React.Component {
           <div className={this.props.displayedLabClass || 'my-2'}>
             <div className="first-positive-lab-result-header pb-1">
               <span className="input-label">FIRST POSITIVE LAB RESULT</span>
-              <div className="edit-link">
+              <span data-for="delete-first_positive_lab_tooltip" data-tip="" className="edit-link">
```

Comment:
can we make this all `-` instead of a mix between `-` and `_`

Reply:
[9d0764e](https://github.com/SaraAlert/SaraAlert/pull/1205/commits/9d0764e18fcec02f59b0d8ce888ddf4e029a0e36)